### ISSUE TITLE
SimpleSource for 702

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#139]
see https://github.com/larshp/abapGit/pull/602

**Comment by EduardoCopat** [comment]
I'm sorry, but what would this Simple Source be?

**Comment by larshp** [comment]
the data sources are usually https://larshp.github.io/abaplint/viz/reuse_source.svg

however it allows too much in a 702 context, so need a new "SimpleSource" class which contains the constructs allowed for 702

sorry about the issue description :o)

**Comment by larshp** [comment]
```abap
READ TABLE mo_repo->get_files_local( ) INTO ls_local_file WITH KEY file = is_file.
```

**Comment by larshp** [comment]
https://syntax.abaplint.org/?filter=source#/expression/BasicSource

**Comment by larshp** [comment]
fixed in 2.57.0
